SimplenoteWidgets/Tools/DemoContent.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+struct DemoContent {
+    static let singleNoteTitle = "Twelve Tone Serialism"
+    static let singleNoteContent = "The twelve-tone technique is often closely related with the compositional style, serialism. Fundamentally, twelve-tone serialism is a compositional technique where all 12 notes of the chromatic scale are played with equal frequency throughout the piece without any emphasis on any one note. For this reason, twelve-tone serialism avoids being in any key. Arnold Schoenberg was arguably the most influential composers who embraced the twelve-tone technique. Schoenberg described the system as a “Method of composing with twelve tones which are related only with one another.”"
+    static let singleNoteContentAlt = "- [] Check item one \n- [] check item two\n- [] check item three"
```

Comment:
Localized?

Reply:
I'm not sure if this needs to be localized.  _**Most**_ of what this does is power the SwiftUI previews. 

I was thinking that I would remove all the UI previews, but we would still need this demo content because it servers a few other purposes.  It gives the widget placeholder some sort of shape if the placeholder appears, and if for some reason there is a failure fetching the user's most recent note then this text will appear.  My feeling is that we don't really need this to be localized, WDYT?

Reply:
TBH if there's a chance for this to be displayed, it should be localized! (Rule of thumb is: any string that may go to screen, NSLocalized!)

Localizations are extracted by the `Scripts/localize.py`. You'll prob. neeed to patch [this spot, exactly](https://github.com/Automattic/simplenote-ios/blob/develop/Scripts/localize.py#L124) as follows:

```
    genstrings_cmd = 'genstrings -q -o "%s" `find ./Simplenote ./SimplenoteShare ./SimplenoteWidgets -name "*.m" -o -name "*.swift"`'
```

(Otherwise SimplenoteWidgets won't be picked up). Same for SimplenoteShare!!

Reply:
I wonder if we should use a different text for this demo content then.  I had been using the text out of the zeplin designs just as something to put in there, but it might make sense to actually consider what this will say.

@SylvesterWilmott can you weigh in on what you would want as far as copy for placeholder text for the widgets?  There are 3 texts I can think of that may appear to users.
Title for the note widget
body for the note widget
8 note titles for the list widget

Currently all of these are using the zeplin text, which may not translate (Like I don't know how well Twelve Tone Serialism will read to users translated into another language?)  But we can continue to use these strings if you prefer